rdmo/management/assets/js/reducers/importsReducer.js

```diff
@@ -48,6 +48,9 @@ export default function importsReducer(state = initialState, action) {
       if (index > -1) {
         const elements = [...state.elements]
         elements[index] = {...elements[index], ...action.values}
+        if (elements[index].model === 'domain.attribute') {
+          elements[index].path = buildPath(elements[index].key, elements[index].parent ? elements[index].parent.uri : null, elements)
```

Comment:
maybe `buildPath` should be called `buildPathForAttribute`  when it's only used for the attributes?